metadata-ingestion/src/datahub/ingestion/source/iceberg/iceberg_common.py

```diff
@@ -184,7 +223,7 @@ def get_paths(self) -> Iterable[Tuple[str, str]]:
         if self.adls:
             yield from self._get_adls_paths(self.adls.base_path, 0)
         elif self.localfs:
-            yield from self._get_localfs_paths(self.localfs, 0)
+            yield from self._get_localfs_paths(self.localfs[len("file:"):], 0)
```

Comment:
I think the following is safer: 
```suggestion
            yield from self._get_localfs_paths(self.localfs.ltrim('file:'), 0)
```